From: Frederic Weisbecker <fweisbec@gmail.com>
Date: Sun, 17 Jan 2010 10:36:08 +0100
Subject: [PATCH 090/279] perf: Make __perf_event_sched_out static
Git-commit: 42cce92f4ddfa41e2dfe26fdcad4887943c032f2
Patch-mainline: v2.6.34-rc1
References: FATE#311392, BNC#685313
Signed-off-by: Tony Jones <tonyj@suse.de>

__perf_event_sched_out doesn't need to be globally available, make
it static.

Signed-off-by: Frederic Weisbecker <fweisbec@gmail.com>
Cc: Peter Zijlstra <peterz@infradead.org>
Cc: Paul Mackerras <paulus@samba.org>
Cc: Ingo Molnar <mingo@elte.hu>
Cc: Arnaldo Carvalho de Melo <acme@infradead.org>
Signed-off-by: Robert Richter <robert.richter@amd.com>
---
 kernel/perf_event.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/kernel/perf_event.c b/kernel/perf_event.c
index 9b0c62e..41f10c5 100644
--- a/kernel/perf_event.c
+++ b/kernel/perf_event.c
@@ -1050,8 +1050,8 @@ static int perf_event_refresh(struct perf_event *event, int refresh)
 	return 0;
 }
 
-void __perf_event_sched_out(struct perf_event_context *ctx,
-			      struct perf_cpu_context *cpuctx)
+static void __perf_event_sched_out(struct perf_event_context *ctx,
+				   struct perf_cpu_context *cpuctx)
 {
 	struct perf_event *event;
 
-- 
1.7.3.4

